numpy/lib/function_base.py

```diff
@@ -1975,28 +1975,27 @@ def corrcoef(x, y=None, rowvar=1, bias=0, ddof=None):
         variable, with observations in the columns. Otherwise, the relationship
         is transposed: each column represents a variable, while the rows
         contain observations.
-    bias : int, optional
```

Comment:
Should probably keep the documentation for both of these, but just say they have no affect and are deprecated. Like so
```
    bias :
        .. deprecated:: 1.10.0
        Has no affect, not used.
```